crates/stackable-operator/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
-
 ## [Unreleased]
 
+### Added
+
+- Added cert lifetime setter to `SecretOperatorVolumeSourceBuilder` ([#915])
+
+All notable changes to this project will be documented in this file.
```

Comment:
I think this line got accidentally moved here. It should be reverted.

Reply:
Oops! Fixed.